dask/array/svg.py

```diff
@@ -2,10 +2,12 @@
 
 import math
 import re
+from functools import lru_cache
 
 import numpy as np
 
 
+@lru_cache(maxsize=512)
```

Comment:
Definitely in favor of caching but I'm a little worried about how large these SVGs might get. That's more a function of not knowing much about the outputs than it is having seen large SVGs in practice. @dcherian do have an example of a particularly bad example where you want caching?  

Also, why `maxsize=512` instead of the default `128`? I could be off, but 128 seems sufficient for how many times I imagine this function is called per Python session